.activate-venv.sh

```diff
@@ -13,12 +13,18 @@ else
     echo "* the no-delete-venv flag is set. preserving venv"
 fi
 
-if [ ! -d venv ]; then
-    # build venv if one doesn't exist
-    virtualenv --python=`which python2` venv
+python=/usr/bin/python3.5
+py=${python##*/} # ll: python3.5
+
+# build venv if one doesn't exist OR 
+# venv exists but the right python isn't installed
+if [ ! -e "venv/bin/$py" ]; then
+    echo "could not find venv/bin/$py, recreating venv"
+    rm -rf venv
+    $python -m venv venv
```

Comment:
is this substituting `virtualenv --python=... venv`?

Reply:
yup, virtualenv is incorporated into python3 so no need to install it separately. 

python3 ships with a `venv` tool, so you could just do `venv myvenv`, but that has been deprecated so it's now as you see above: `python -m venv myvenv`

https://docs.python.org/3/library/venv.html